### PR TITLE
Setup Index Page & Ran Migration Stations Table & Setup Stations Model

### PR DESCRIPTION
**Pull request by cooperdutfield**
Created the Index Page w/ migrations & model. Connected it with the backend.  Updated the readme slightly still needs work.
 https://www.loom.com/share/4dbb5e09f164425d8ed95f7052978508

### CODE REVIEW

File: client/src/components/StationsIndex.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const stationIndexItems = stations.map(stationItem => {
+    return (
+        <div>
+           <h1>{stationItem.name}</h1>
+           <h1>{stationItem.line}</h1>
+           <h1>{stationItem.location}</h1>
+        </div>
+    )
+})
```
Comment (dovakeen118) [inline client/src/components/StationsIndex.js:33]: lines 25-33 should be indented to be within the StationIndex component (similar indentation as the useEffect, fetch request, etc)

File: client/src/components/StationsIndex.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        getStations()
+    }, [])
+
+const stationIndexItems = stations.map(stationItem => {
+    return (
+        <div>
+           <h1>{stationItem.name}</h1>
+           <h1>{stationItem.line}</h1>
+           <h1>{stationItem.location}</h1>
+        </div>
```
Comment (dovakeen118) [inline client/src/components/StationsIndex.js:31]: consider moving this to its own component. There should also only ever be 1 `h1` element on a given webpage
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page

File: client/src/components/StationsIndex.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    )
+})
+
+return (
+    <div>
+        <h1> {stationIndexItems} </h1>
```
Comment (dovakeen118) [inline client/src/components/StationsIndex.js:37]: the `stationIndexItems` are a collection of `div`s meaning that wrapping them in this `h1` is improper HTML nesting

File: client/src/components/StationsIndex.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <h1> Boston T Stations </h1>
```
Comment (dovakeen118) [inline client/src/components/StationsIndex.js:38]: might consider moving this header above the list of items so that users see it first before any info

**Review by dovakeen118 — COMMENTED** [review]
looking good, some small updates to be made!

**Comment by cooperdutfield** [comment]
Leaving map for right now going to make it into its own tile component later.

File: client/src/components/StationsIndex.js [hunk 5]
```diff
@@ -22,22 +22,22 @@ const StationIndex = props => {
...
+        )
+    })
+
     return (
         <div>
-           <h1>{stationItem.name}</h1>
-           <h1>{stationItem.line}</h1>
-           <h1>{stationItem.location}</h1>
+            <h1>Boston T Stations</h1>
+            <ul>{stationIndexItems}</ul>
```
Comment (dovakeen118) [inline client/src/components/StationsIndex.js:38]: this is _okay_, but an `ul` is going to expect `li` elements as its children, but `stationIndexItems` is a collection of `div`'s. Consider correcting this when creating individual components for each station.

**Pull request merged**